src/components/page/common/Fallback/fallback.view.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+import browser from "webextension-polyfill";
+
+import { Text } from "@arconnect/components";
+
+export const FallbackView = () => {
+  return (
+    <DivWrapper>
+      <Text heading>{browser.i18n.getMessage("fallback")}</Text>
+    </DivWrapper>
```

Comment:
If `process.env.NODE_ENV === "development"` only, could you add a second line with details about the error: name, message, stack.